setting-up-webauthn-mfa.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: default
+title: Setting up WebAuthn / Passkey multi-factor authentication
+url: /setting-up-webauthn-mfa
+previous: /setting-up-multifactor-authentication
+next: /setting-up-otp-mfa
+---
+
+<em class="t-gray">Setting up WebAuthn multifactor authentication</em>
+
+Using WebAuthn for multifactor authentication (MFA) is the best way to
+protect your account from takeover. It's stronger and easier to use than
+[OTP codes](/setting-up-otp-mfa).
+
+## Prerequisite
+
+To use WebAuthn, you will need at least _one_ of the following:
+
+* A hardware security token (sometimes called a security key), such as a
+  YubiKey or Google Titan Key.
+* A built-in hardware device, such as TouchID, FaceID or Windows Hello.
+* A browser that supports the "Passkey" standard. Up-to-date versions of
+  Chrome, Safari, Firefox and Edge all support this standard.
+
+Unfortunately implementations of these experiences vary, so we can't show
+the exact details, but we will point out the steps that are specific to
+using rubygems.org.
+
+## Enabling WebAuthn multifactor authentication
```

Comment:
```suggestion
## Enabling WebAuthn multi-factor authentication
```
There's some inconsistencies between mutifactor vs multi-factor in the docs (eg. https://github.com/rubygems/guides/pull/332/files#diff-ec662ede6bef96bdfc86b706810db57da07dbfee38d8bbe74fd8bedad68b8604R3). I prefer having the hyphened version.